### DISCUSSION TITLE
RoderickShao_notebook/%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9F%BA%E7%A1%80%E8%AF%BE/%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%BB%84%E6%88%90/Chapter%202/

### DISCUSSION THREAD

**Discussion by giscus** [RoderickShao/RoderickShao_notebook#4] [Announcements]
# RoderickShao_notebook/%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9F%BA%E7%A1%80%E8%AF%BE/%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%BB%84%E6%88%90/Chapter%202/

RoderickShao's NoteBook

https://roderickshao.github.io/RoderickShao_notebook/%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9F%BA%E7%A1%80%E8%AF%BE/%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%BB%84%E6%88%90/Chapter%202/

**Comment by Provence-24** [comment]
In 2.7.2 RISC-V assembly code:把寄存器 “的值“ 存在sp中可能更准确

**Comment by Provence-24** [comment]
2.7.2 递归自己原创的那一段代码中 blt是小于。所以把x10与0用blt比较，则再0的时候会继续做，这样返回时第一个sp里面的值是0，不是1。这样乘起来会有点问题。

**Reply by Provence-24** [reply]
应该用ble

**Reply by RoderickShao** [reply]
好的，我改好了
